### ISSUE TITLE
PHP8 deprecating warning

### ISSUE THREAD

**Issue by theblindfrog** [ExpDev07/laravel-cashier-stripe-connect#10]
Hi there, 

I'm getting this warning when running on PHP8

`Deprecated: Required parameter $amount follows optional parameter $refundFee in /Users/Code/project/vendor/expdev07/laravel-cashier-stripe-connect/src/Concerns/ManagesTransfer.php on line 53`

As far as I can tell, this is due to PHP 8 requiring required arg first, before optional items [per this article](https://php.watch/versions/8.0/deprecate-required-param-after-optional)

```php
public function reverseTransferFromStripeAccount(Transfer $transfer, $refundFee = false, ?int $amount, array $options = []): TransferReverse
```

could be altered to
```php
public function reverseTransferFromStripeAccount(Transfer $transfer, ?int $amount, $refundFee = false, array $options = []): TransferReverse
```

but this is a breaking change, I'd imagine

**Comment by ExpDev07** [comment]
Could u fix this in ur most resent PR? Thanks :)

**Comment by theblindfrog** [comment]
Can do!

**Comment by ExpDev07** [comment]
Yes, let's just keep this issue open then, and not do anything about it right now :).
